pkg/cmd/cli/cmd/login.go

```diff
@@ -20,9 +20,9 @@ import (
 
 const (
 	loginLong = `
-Log in to an OpenShift server and save login for subsequent use
+Log in to a Kubernetes server and save login for subsequent use
```

Comment:
I'll change it to "Log in to your server ..."

On Thu, Jul 23, 2015 at 3:21 PM, Jessica Forrester <notifications@github.com
> wrote:

> In pkg/cmd/cli/cmd/login.go
> <https://github.com/openshift/origin/pull/3879#discussion_r35360654>:
>
> > @@ -20,9 +20,9 @@ import (
> >
> >  const (
> >  	loginLong = `
> > -Log in to an OpenShift server and save login for subsequent use
> > +Log in to a Kubernetes server and save login for subsequent use
>
> is this going to be confusing to end users that dont know openshift is
> built on kubernetes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3879/files#r35360654>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift